docs/index.html

```diff
@@ -2,11 +2,11 @@
     <head>
         <meta name="viewport" content="width=device-width">
         <link rel="stylesheet" href='./styles.css' />
-        <link rel="stylesheet" href='https://cdn01.boxcdn.net/platform/preview/1.58.2/en-US/preview.css' />
-        <link rel="stylesheet" href='https://unpkg.com/box-annotations@3.3.0/lib/annotations.css' />
```

Comment:
I was looking at https://unpkg.com/#/ and it seems to suggest we can fetch the latest version of the npm package by not specifying the version. Would it be beneficial to do that for both preview & annotations so this file doesn't need to get updated?

Reply:
yeah! We can't do that for preview because it's not on NPM, but I'll do that for annotations.

Reply:
oh that's sad, then we still need to do a preview update. Do we have control over the boxcdn to create a `latest` directory that points to the latest?